### ISSUE TITLE
Multiply U and V mask by ID mask while training

### ISSUE THREAD

**Issue by yshah43** [yshah43/DPOD#5]
ID masks are not binary. 
Using torch.where to convert the ID mask to binary would add a discontinuity and thus, make it non-differentiable.

**Comment by yshah43** [comment]
@suyz526 - do you know a way around this?

**Comment by suyz526** [comment]
id mask divided by itself, then all the id will be 1. 

But I don't know what will happen with 0/0.  

I see another implementation of DPOD, I will have a look there.
